src/webgl/framebuffer.js

```diff
@@ -8,6 +8,24 @@ import {getTypedArrayFromGLType, getGLTypeFromTypedArray} from '../utils/typed-a
 import {log} from '../utils';
 import assert from 'assert';
 
+// Local constants - will collapse during minification
+const GL_FRAMEBUFFER = 0x8D40;
```

Comment:
sorry for the confusion but you did answer my question. So it's for reduction of the library size then.